examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -191,12 +191,16 @@ CHIP_ERROR AppTask::Init()
     SetDeviceInstanceInfoProvider(&mFactoryDataProvider);
     SetDeviceAttestationCredentialsProvider(&mFactoryDataProvider);
     SetCommissionableDataProvider(&mFactoryDataProvider);
+    // Read EnableKey from the factory data.
+    MutableByteSpan enableKey(sTestEventTriggerEnableKey);
+    err = mFactoryDataProvider.GetEnableKey(enableKey);
```

Comment:
I think we shouldn't initialize the delegate if this fails. Alternatively, initialize `sTestEventTriggerEnableKey` to all zeros if `CONFIG_CHIP_FACTORY_DATA` and ignore an error here (assignment to `err` is not needed if we don't check the value).

Reply:
I just forgot to add checking here :/